### PR TITLE
Dashboard copy and successful nominee page

### PR DESCRIPTION
**Pull request by hammerfunk**

### CODE REVIEW

File: app/forms/qae_2014_forms/innovation/innovation_step4.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def innovation_step4
               If you can give quantitative evidence of your initiatives/improvement/success, then do so.
               </p>
             <p>
-              If you have too many initiatives, just outline the ones you think are most relevant/important.
+              If you have too many initiatives, just outline the ones you think are most relevant/important you think are most relevant/important.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step4.rb]: Line is too long. [146/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step4.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@ def trade_step4
               If you can give quantitative evidence of your initiatives/improvement/success, then do so.
               </p>
             <p>
-              If you have too many initiatives, just outline the ones you think are most relevant/important.
+              If you have too many initiatives, just outline the ones you think are most relevant/important you think are most relevant/important.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step4.rb]: Line is too long. [146/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb [hunk 3]
```diff
@@ -27,7 +27,7 @@ def development_step4
               If you can give quantitative evidence of your initiatives/improvement/success, then do so.
               </p>
             <p>
-              If you have too many initiatives, just outline the ones you think are most relevant/important.
+              If you have too many initiatives, just outline the ones you think are most relevant/important you think are most relevant/important.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step4.rb]: Line is too long. [146/100]

File: config/routes.rb [hunk 4]
```diff
@@ -50,6 +50,7 @@
   get "/award_winners_section"                          => "content_only#award_winners_section",                          as: "award_winners_section"
   get "/press_comment"                                  => "content_only#press_comment",                                  as: "press_comment"
   get "/buckingham_palace_attendance"                   => "content_only#buckingham_palace_attendance",                   as: "buckingham_palace_attendance"
+  get "/submitted_nomination_successful"                => "content_only#submitted_nomination_successful",                as: "submitted_nomination_successful"
```
Comment (matthewford) [inline config/routes.rb]: Line is too long. [159/100]<br>Unnecessary spacing detected.

File: app/views/content_only/declaration_of_corporate_responsibility.html.slim [hunk 5]
```diff
@@ -87,6 +88,14 @@ form
               textarea.js-char-count data-word-max="500" rows="5"
               .clear
 
+          br
+
+          .application-notice.info-notice
+            p
+              ' You may still edit your submitted Declaration of Corporate Responsibility up until the deadline on
+              = Settings.current.deadlines.where(kind: "audit_certificates").first.decorate.formatted_trigger_time
```
Comment (jaleszek) [inline app/views/content_only/declaration_of_corporate_responsibility.html.slim]: I would wrap with any method (in Settings class?), so many `.` in many places pretty hurts xD

Comment (jaleszek) [inline app/views/content_only/declaration_of_corporate_responsibility.html.slim]: or helper meth

Comment (hammerfunk) [inline app/views/content_only/declaration_of_corporate_responsibility.html.slim]: lol sorry I'll try that

File: app/controllers/application_controller.rb [hunk 6]
```diff
@@ -113,4 +113,10 @@ def submission_deadline
     @submission_deadline ||= settings.deadlines.where(kind: "submission_end").first
   end
   helper_method :submission_deadline
+
+  def application_deadline(kind)
+    settings.deadlines.where(kind: kind).first.decorate.formatted_trigger_time
+  end
+  helper_method :application_deadline
+
```
Comment (matthewford) [inline app/controllers/application_controller.rb]: Extra empty line detected at class body end.

**Pull request merged**